examples/binary_examples/PyFstat_example_binary_mcmc_vs_grid_comparison.py

```diff
@@ -0,0 +1,296 @@
+#!/usr/bin/env python
+
+import pyfstat
+import os
+import numpy as np
+import matplotlib.pyplot as plt
+
+# Set to false to include eccentricity
+circular_orbit = False
+
+label = os.path.splitext(os.path.basename(__file__))[0]
+outdir = os.path.join("PyFstat_example_data", label)
+
+# Parameters to generate a data set
+data_parameters = {
+    "sqrtSX": 1e-22,
+    "tstart": 1000000000,
+    "duration": 60 * 86400,
```

Comment:
may going longer break some of the degeneracies seen in plots?

Reply:
That depends on how much time we want to wait: Using the current set up, it already takes ~5 minutes.

Maybe we can increase length and reduce the number of MCMC steps?

Reply:
I achieved similar results with a shorter setup.